<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hellow WIdget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
         background-color:rgb(68,68,68);
         color:rgb(255,255,255);
         font:10pt &quot;幼圆&quot;;
}
QMenuBar{
         background-color:rgb(200,200,200);
         color:rgb(60,60,60);
}
QWidget:checked{
         background-color:rgb(150, 150, 150);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="LCOpenGLWidget" name="openGLWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查看</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actDrawRect"/>
   <addaction name="actClear"/>
   <addaction name="actWireframe"/>
  </widget>
  <action name="actDrawRect">
   <property name="text">
    <string>画一个矩形</string>
   </property>
  </action>
  <action name="actClear">
   <property name="text">
    <string>清空图形</string>
   </property>
  </action>
  <action name="actWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>线框渲染</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LCOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">lcopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
